features/dd-sdk-android-trace/src/test/kotlin/com/datadog/android/trace/OtelTracerBuilderProviderTest.kt

```diff
@@ -577,4 +607,225 @@ internal class OtelTracerBuilderProviderTest {
     }
 
     // endregion
+
+    // region bundle with RUM
+
+    @Test
+    fun `M build a Span with RUM context W buildSpan`() {
```

Comment:
same applies to the tests below

```suggestion
    fun `M build a Span with RUM context W startSpan`() {
```